rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -449,3 +451,23 @@ def test_force_delete_used_blueprints_via_import(self):
                           first_blueprint_id)
 
         self.client.blueprints.delete(first_blueprint_id, True)
+
+    @attr(client_min_version=2,
+          client_max_version=base_test.LATEST_API_VERSION)
+    def test_not_listing_hidden_blueprints(self):
+        blueprint_file = 'blueprint_with_inputs.yaml'
+        blueprint_path = os.path.join(
+            self.get_blueprint_path('mock_blueprint'),
+            blueprint_file)
+        b0_id = 'b0'
+        b1_id = 'b1'
+        self.client.blueprints.upload(blueprint_path, b0_id)
```

Comment:
there is a way to send the HTTP req without the client, so I think it's a better way to go because it will test the is_hidden assignment